src/main/java/me/leoko/advancedban/bungee/BungeeMethods.java

```diff
@@ -144,12 +147,23 @@ public void sendMessage(Object player, String msg) {
 
     @Override
     public boolean hasPerms(Object player, String perms) {
-        return ((CommandSender) player).hasPermission(perms);
+        try {
+            return ((CommandSender) player).hasPermission(perms);
+        } catch (Exception ex) {
```

Comment:
Okay then add a not null check here or in the PubSubListener but just catching **all** exceptions is not a good solution for this ;)